logging/tests/unit/test_logger.py

```diff
@@ -189,7 +191,8 @@ def test_log_text_w_unicode_explicit_client_labels_severity_httpreq(self):
                                 labels=DEFAULT_LABELS)
 
         logger.log_text(TEXT, client=client2, labels=LABELS,
-                        insert_id=IID, severity=SEVERITY, http_request=REQUEST)
+                        insert_id=IID, severity=SEVERITY, http_request=REQUEST,
+                        trace=TRACE)
```

Comment:
It seems like `Logger.log_struct` and `Logger.log_proto` should also have a testcase exercising the `trace` argument (LL 243, 358 below).

Reply:
added `_trace` to

```
test_log_text_w_unicode_explicit_client_labels_severity_httpreq_trace()
test_log_struct_w_explicit_client_labels_severity_httpreq_trace()
test_log_proto_w_explicit_client_labels_severity_httpreq_trace()
```